tests/test_models.py

```diff
@@ -488,15 +502,18 @@ def test_pipeline_databuilder_specifies_different_output():
         Pipeline(**data)
 
 
-def test_pipeline_databuilder_specifies_same_output():
+def test_pipeline_databuilder_recognizes_old_action_spelling():
+    # The action name is used to select the validator, so the only way to know that it's been recognized is
+    # to give it an invalid input and check that validation fails.
     data = {
         "version": 1,
         "actions": {
-            "generate_dataset": {
-                "run": "databuilder:latest generate_dataset --output=output/dataset.csv",
+            "generate-dataset": {
```

Comment:
Shouldn't this stay the same so that we're still testing the old name?

Reply:
I've added a dedicated test for the old name rather than duplicating all of the tests.